### PR TITLE
Feature/changelog-updates

### PR DESCRIPTION
**Pull request by Surxe**
Corrects {Template} syntax to {{Template}} in changelog parser

### CODE REVIEW

File: src/parser/parsers/changelogs.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def _embed_icons(self, changelog):
 
                         if resource_name not in log:
                             continue
-                        icon = f'{{AbilityIcon|Ability={resource_name}|Size=20}}'
+                        icon = "{{AbilityIcon|Ability="+resource_name+"|Size=20}}"
```
Comment (hassayag) [inline src/parser/parsers/changelogs.py:95]: The format of ability icon looks to be `{{AbilityIcon|Siphon Life|Abrams}}`

**Review by hassayag — CHANGES_REQUESTED** [review]

**Comment by Surxe** [comment]
Taking the opportunity to transition this to a restructuring of changelog parser; feature may be more descriptive than bugfix.

**Review by hassayag — COMMENTED** [review]
If you put one of these changelogs into the wiki, does it render what you would expect with icons, links, etc.

**Comment by Surxe** [comment]
> If you put one of these changelogs into the wiki, does it render what you would expect with icons, links, etc.

not yet

**Comment by B13rg** [comment]
Okay, squashed the develop branch on and cleaned up my changes.
I tried running a generate, but I hit an error.  I have the same error on develop, so my test is nothing to go by

error:
```
File "/repo/src/parser/parsers/ability_ui.py", line 48, in run
deadbot-1  |     raise Exception(f'[ERROR] Failed to parse ui for ability {ability["Key"]}', e)
deadbot-1  | Exception: ('[ERROR] Failed to parse ui for ability ability_nano_shadow_step', Exception('No localized string for key MeleeAttackSpeedBonus_label')
```

**Comment by B13rg** [comment]
I can test with just doing changelogs, got it to generate!

**Comment by Surxe** [comment]
Closing. #84 is opening in its place

**Pull request closed without merging**